src/breadcrumb/types.ts

```diff
@@ -1,8 +1,13 @@
 export type FadePosition = "left" | "right" | "both";
 
+export interface FadeColorSet {
+    left?: string[] | undefined;
+    right?: string[] | undefined;
+}
+
 export interface BreadcrumbProps {
     links: React.AnchorHTMLAttributes<HTMLAnchorElement>[];
-    fadeColor?: string[] | undefined;
+    fadeColor?: string[] | FadeColorSet | undefined;
```

Comment:
alternatively we can make the typing "enforce" at least one element `[string, ...string[]]`

Reply:
You mean to counter not having to check for length?